utils.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	errCannotDecryptReponse = Error("cannot decrypt response")
-	errCouldNotAllocateIP   = Error("could not find any suitable IP")
+	ErrCannotDecryptReponse = Error("cannot decrypt response")
```

Comment:
If these will be exported should we also fix this typo to `ErrCannotDecryptResponse`?

Reply:
Done